oddt/virtualscreening.py

```diff
@@ -304,6 +304,20 @@ def score(self, function, protein=None, *args, **kwargs):
                         new_kwargs['pdbbind_version'] = int(bit.replace('pdbbind', ''))
                 sf = nnscore.load(**new_kwargs)
                 sf.set_protein(protein)
+            elif function.lower().startswith('plec'):
+                from oddt.scoring.functions.PLECscore import PLECscore
+                new_kwargs = {}
+                for bit in function.lower().split('_'):
+                    if bit.startswith('pdbbind'):
```

Comment:
1. It seems that fingerprint size is also coded in the file name. Why we don't use it here?

2. I think it would be better to use `re.match` here. Now, if we change the naming scheme or break something in another way, we might not even notice.

We can even use a single expression to get all params, something like
```python
re.match('plec([^_]+)_p([0-9]+)_l([0-9]+)_pdbbind([0-9]{4})_s([0-9]+)', name).groups()
```

Reply:
Oh, that's cool, I'll do it. It has iteratively grown to such monster.

Reply:
On the other hand, we could not tackle it with one regex. It is a feature that you can parially define the sf, like `pleclinear_s1024`, and take the other param's default values. We could tackle this issue in the future, but I think it is out of scope of this PR.